protocols/relay/Cargo.toml

```diff
@@ -30,6 +30,8 @@ prost-build = "0.7"
 
 [dev-dependencies]
 env_logger = "0.8.3"
+hex = "0.4.2"
```

Comment:
Where is this used?

Reply:
You right, I forgot this dependency after stop to use this crate.